hash.c

```diff
@@ -3804,21 +3804,25 @@ hash_equal(VALUE hash1, VALUE hash2, int eql)
 
 /*
  *  call-seq:
- *    hash == object -> true or false
+ *    self == object -> true or false
  *
- *  Returns +true+ if all of the following are true:
- *  * +object+ is a +Hash+ object.
- *  * +hash+ and +object+ have the same keys (regardless of order).
- *  * For each key +key+, <tt>hash[key] == object[key]</tt>.
```

Comment:
This new documentation hides this important information in a link to the new document. IMO this is important and so we should make this easy to access.

Reply:
Fixed (I think).